matrix_client/client.py

```diff
@@ -111,7 +111,8 @@ def global_callback(incoming_event):
 
     def __init__(self, base_url, token=None, user_id=None,
                  valid_cert_check=True, sync_filter_limit=20,
-                 cache_level=CACHE.ALL, encryption=False, encryption_conf=None):
+                 cache_level=CACHE.ALL, encryption=False, encryption_conf=None,
+                 api_connection_timeout=60):
```

Comment:
Can we just call this `timeout` instead of `connection_timeout` or `api_connection_timeout` for conciseness?